applications/nrf5340_audio/src/modules/lc3_streamer.c

```diff
@@ -369,8 +374,7 @@ int lc3_streamer_stream_register(const char *const filename, uint8_t *const stre
 		return ret;
 	}
 
-	strncpy(streams[*streamer_idx].filename, filename,
-		ARRAY_SIZE(streams[*streamer_idx].filename));
+	strcpy(streams[*streamer_idx].filename, filename);
```

Comment:
Above, where we check that `strlen(filename) > CONFIG_FS_FATFS_MAX_LFN - 1` we should rather check against `ARRAY_SIZE(streams[*streamer_idx].filename)`